docs/index.bs

```diff
@@ -3154,7 +3157,28 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
                       1. If |client| is not null, |response|'s [=response/type=] is "`opaque`", and [=cross-origin resource policy check=] with |request|'s [=request/origin=], |client|, "", and |response|'s [=filtered response/internal response=] returns <b>blocked</b>, then return null.
                       1. Return |response|.
               1. Return null.
-      1. If |request| is a <a>non-subresource request</a>, |request| is a [=navigation request=], |registration|'s [=navigation preload enabled flag=] is set, |request|'s [=request/method=] is \`<code>GET</code>\`, |registration|'s [=active worker=]'s [=set of event types to handle=] [=set/contains=] <code>fetch</code>, and |registration|'s [=active worker=]'s [=all fetch listeners are empty flag=] is not set then:
+          1. Else if |source| is {{RouterSourceEnum/"race-network-and-fetch-handler"}}, and |request|'s [=request/method=] is \`<code>GET</code>\` then:
+              1. Let |queue| be an empty [=queue=] of [=/response=].
+              1. Let |raceFetchController| be null.
+              1. Set |raceResponse|'s [=race response/value=] to "<code>pending</code>".
+              1. [=map/Set=] |activeWorker|'s [=service worker/global object=]'s [=race response map=][|request|] to |raceResponse|.
```

Comment:
It might be simpler if the race response map is associates with activeWorker (i.e. a service worker) rather than its global scope? Especially as at this point in the algorithm the worker hasn't been started yet, so global object might very well be null.

Reply:
That's a good point. How about keep the global scope having the map, and move this [=map/set=] to the step 16.2 in _[Create Fetch Event and Dispatch](https://pr-preview.s3.amazonaws.com/sisidovski/ServiceWorkerForStaticRoutingAPI/pull/10.html#create-fetch-event-and-dispatch-algorithm)_? At this point, _[Run Service Worker](https://pr-preview.s3.amazonaws.com/sisidovski/ServiceWorkerForStaticRoutingAPI/pull/10.html#run-service-worker)_ was already called to we can expect the global object is ready.

Reply:
Updated as above. Let me know if this doesn't solve your concern.